go.mod

```diff
@@ -5,6 +5,9 @@ go 1.15
 require (
 	github.com/bloxapp/eth2-key-manager v1.0.6
 	github.com/dgraph-io/badger/v3 v3.2011.1
+	github.com/docker/go-connections v0.4.0 // indirect
```

Comment:
Do we use these packages?

Reply:
seems we use it indirectly.. I didn't add it